app/controllers/operationsAndFunds/BankDetailsController.scala

```diff
@@ -48,19 +49,26 @@ class BankDetailsController @Inject()(
   val sectionName: String = "operationsAndFunds.section"
 
   def onPageLoad(mode: Mode): Action[AnyContent] = (identify andThen getData andThen requireData).async { implicit request =>
-    request.userAnswers.get(CharityNamePage) match {
-      case Some(charityName) =>
 
-        val form: Form[BankDetails] = formProvider(messagePrefix, charityName.fullName)
+    if (request.userAnswers.get(Section1Page).contains(false)){
```

Comment:
redirecting to service reset page if SectionPage1 is not defined... Please fix this